images/base/files/usr/local/bin/entrypoint

```diff
@@ -368,10 +386,17 @@ enable_network_magic(){
         echo "ERROR: Have an old IPv4 address but no current IPv4 address (!)" >&2
         exit 1
       fi
-      # kubernetes manifests are only present on control-plane nodes
-      sed -i "s#${old_ipv4}#${curr_ipv4}#" /etc/kubernetes/manifests/*.yaml || true
-      # this is no longer required with autodiscovery
-      sed -i "s#${old_ipv4}#${curr_ipv4}#" /var/lib/kubelet/kubeadm-flags.env || true
+      if [[ "${old_ipv4}" != "${curr_ipv4}" ]]; then
+        # kubernetes manifests are only present on control-plane nodes
+        sed -i "s#${old_ipv4}#${curr_ipv4}#" /etc/kubernetes/manifests/*.yaml || true
```

Comment:
It could work but it would be a change irrelevant to the PR's purpose as the script has been using sed from the beginning and `kubeadm.conf`, `kubeadm-flags.env` will still need sed. It may require a few more lines to check if this is control-plane node before invoking `kubeadm init phase control-plane`. 
Do we want to make the change in this PR?

Reply:
Sed means we retain any user changes performed manually out of band except the targeted edit we're making.

Reply:
thanks for commenting, then I suppose we could keep sed

Reply:
we should however, do some fixes so we replace the actual literals instead of letting the IP address be mis-interpreted as a regex, though that bug is not new to this PR.

Reply:
rebased the PR (because it's based on out of date version of entrypoint script) and then fixed this, we regex escape the ip addresses now (I think ...)